web-ui/src/app/app.component.ts

```diff
@@ -120,6 +120,13 @@ export class AppComponent implements OnInit {
       'label.normalFee': 'Normal',
       'label.lowFee': 'Low',
       'label.satoshis': 'Satoshis',
+      'label.merchantAddress': 'Merchant Address',
+      'label.merchantStakeAddress': 'MerchantStake address',
```

Comment:
I guess this is the `Staking address`, which doesn't belong to the merchant but to the staker.